### PR TITLE
Rename to launchpad; Update ecr in travis

### PR DESCRIPTION
**Pull request by ceolaflannery**

### CODE REVIEW

File: WebService.UnitTests/WebService.UnitTests.xproj [hunk 1]
```diff
@@ -14,5 +14,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (jimmyp) [inline WebService.UnitTests/WebService.UnitTests.xproj]: Whats this?

File: Readme.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Run `docker logs microservicebootstrap_workerservice_1` and see "Hello world" to
 3. Create a dotnet core branch - for enthusiasts
 4. Include code coverage & complexity metrics
 
-## Bootstrap idea
+## Template idea
 
 The service is designed in mind considering the Principles of Microservices as documented by Sam Newman.
```
Comment (jimmyp) [inline Readme.md:50]: You could add the logo here! :D

File: .travis.yml [hunk 3]
```diff
@@ -17,9 +17,9 @@ before_install:
```
Comment (jimmyp) [inline .travis.yml]: You've got bad line endings too... my PR #8 added a `.gitattributes` file, maybe that should merge first?

**Comment by jimmyp** [comment]
Build is green 💚  can I get a LGTM?

**Comment by jimmyp** [comment]
LGTM

**Comment by humblelistener** [comment]
@ceolaflannery @jimmyp 
I like the fact bootstrap has been renamed to Template. Happy to rename microservice-bootstrap repo to microservice-template repo.

But naming things to "Launchpad" (tune up project name) is too much.

Can we stick to the name "microservice-template"?

**Pull request merged**